### PR TITLE
[ru] Localize `Infrastructure-as-code`

### LINKED ISSUE #3121: [ru] Localize `Infrastructure as Code (IaC)`
### Description

This Issue is to localize [`Infrastructure as Code (IaC)`](https://github.com/cncf/glossary/blob/main/content/en/infrastructure-as-code.md) into Russian.

### Checklist before opening your PR (put `x` in the checkboxes) — note that you have to _add this checklist to your PR and check all the boxes for it to be approved_.
- [] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.
    
**ETA: TBD**

Useful links:
[Как внести свой вклад](https://glossary.cncf.io/ru/contribute/)
[Руководство по стилю](https://glossary.cncf.io/ru/style-guide/)

[Slack channel](https://app.slack.com/client/T08PSQ7BQ/C05G46RMQTX)
[Telegram channel](https://t.me/ru_cloudnative)

### PR DESCRIPTION
**Pull request by kirkonru**
### Describe your changes
Localize `Infrastructure-as-code`

### Related issue number or link (ex: `resolves #issue-number`)
resolves #3121 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR uses the **dev-ru** branch as a source/target.
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/ru/infrastructure-as-code.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+---
+
+Инфраструктура-как-код — подход, при котором описание инфраструктуры хранится в виде одного или нескольких файлов. 
+Этим он отличается от традиционной модели, в которой инфраструктура как услуга предоставляется вручную, 
+обычно с помощью shell-скриптов или других инструментов конфигурирования.
+
+## Какую проблему решает
+
+Подход к разработке нативных облачных приложений требует, чтобы инфраструктура была одноразовой и воспроизводимой. 
+Также она должна [масштабироваться](/scalability/) по запросу автоматизированным и повторяемым образом, желательно без вмешательства человека.
```
Comment (shurup) [inline content/ru/infrastructure-as-code.md:15]:
```suggestion
Также она должна [масштабироваться](/scalability/) по запросу автоматизированным и воспроизводимым образом, желательно без вмешательства человека. 
```

File: content/ru/infrastructure-as-code.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+Подход к разработке нативных облачных приложений требует, чтобы инфраструктура была одноразовой и воспроизводимой. 
+Также она должна [масштабироваться](/scalability/) по запросу автоматизированным и повторяемым образом, желательно без вмешательства человека. 
+Ручное управление инфраструктурой не удовлетворяет требованиям к скорости и масштабированию, предъявляемым к [нативным облачным приложениям](/ru/cloud-native-apps/). 
+Изменения инфраструктуры, сделанные вручную, не воспроизводимы; они плохо масштабируются и страдают от ошибок в конфигурации.
+
+## Как именно решает проблему
+
+Представляя ресурсы дата-центра (серверы, балансировщики нагрузки, подсети и т. п.) в виде кода, 
+инфраструктурные команды получают единый источник истины для всех конфигураций и возможность 
+управлять центром обработки данных в пайплайнах [CI](/ru/continuous-integration/)/[CD](/ru/continuous-delivery/),
```
Comment (shurup) [inline content/ru/infrastructure-as-code.md:23]:
```suggestion
управлять своим дата-центром прямо в пайплайнах [CI](/ru/continuous-integration/)/[CD](/ru/continuous-delivery/), 
```

File: content/ru/infrastructure-as-code.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+Также она должна [масштабироваться](/scalability/) по запросу автоматизированным и повторяемым образом, желательно без вмешательства человека. 
+Ручное управление инфраструктурой не удовлетворяет требованиям к скорости и масштабированию, предъявляемым к [нативным облачным приложениям](/ru/cloud-native-apps/). 
+Изменения инфраструктуры, сделанные вручную, не воспроизводимы; они плохо масштабируются и страдают от ошибок в конфигурации.
+
+## Как именно решает проблему
+
+Представляя ресурсы дата-центра (серверы, балансировщики нагрузки, подсети и т. п.) в виде кода, 
+инфраструктурные команды получают единый источник истины для всех конфигураций и возможность 
+управлять центром обработки данных в пайплайнах [CI](/ru/continuous-integration/)/[CD](/ru/continuous-delivery/), 
+реализуя контроль версий и стратегии развертывания.
```
Comment (shurup) [inline content/ru/infrastructure-as-code.md:24]:
```suggestion
используя при этом контроль версий и стратегии развертывания.
```

**Review by shurup — APPROVED** [review]
Just a few minor suggestions, which are probably a matter of personal preference.

File: content/ru/infrastructure-as-code.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
+---
+title: Инфраструктура-как-код (IaC)
```
Comment (shurup) [inline content/ru/infrastructure-as-code.md:2]: Should we have hyphens here? They are not used for «Инфраструктура как услуга» (#3226); they are absent in the [Wikipedia article](https://ru.wikipedia.org/wiki/%D0%98%D0%BD%D1%84%D1%80%D0%B0%D1%81%D1%82%D1%80%D1%83%D0%BA%D1%82%D1%83%D1%80%D0%B0_%D0%BA%D0%B0%D0%BA_%D0%BA%D0%BE%D0%B4).

Comment (kirkonru) [inline content/ru/infrastructure-as-code.md:2]:
```suggestion
title: Инфраструктура как код (IaC)
```

**Review by tym83 — APPROVED** [review]

**Pull request merged**